assets/components/organisms/social-share/social-share.scss

```diff
@@ -51,10 +51,28 @@
 }
 
 .social-share-copylink {
-  margin-top: 2rem;
+  display: flex;
+  white-space: nowrap;
+  align-items: center;
+
+  .input-group {
+    margin-left: 0.5rem;
+    padding: 0.2rem;
+    border: 1px solid $gray-200;
+
+    .form-control {
+      border: none !important;
+      box-shadow: none !important;
+      &:before { content: none; }
+    }
+  }
 
   @include media-breakpoint-up(lg) {
     margin-top: 0;
   }
+
+  @include media-breakpoint-down(md) {
```

Comment:
maybe a bit of mobile styling to make it more elegant? 
![capture d ecran 2018-03-29 a 11 41 49](https://user-images.githubusercontent.com/4548432/38082200-4256973a-3346-11e8-92a4-cdd359d9cf4e.png)

Reply:
also the button top-margin seems a bit visually big. Maybe we should diminish it globally? Or maybe it is just that word that makes it look so.

Reply:
Which “top-margin” ? Yes we can to a bit of mobile enhancement, but later. We don't have any mobile design for this component and we are not sure about the amount of social button (G+ is dead...) We will plan a fine tunning round with Guillaume soon 😉

Reply:
I meant button top-padding :D Okay for mobile design